cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -234,14 +231,9 @@ func (sub EthereumSub) Start(completionEvent *sync.WaitGroup) {
 			// The user who starts this must provide a valid last processed block
 			if lastProcessedBlock.Cmp(big.NewInt(0)) == 0 {
 				lastProcessedBlock.Sub(endingBlock, big.NewInt(1))
```

Comment:
In this case, if you want to delete that other logic, if lastProcessedBlock is equal to 0, then you should just set last processed block to the ending block instead of ending block minus one.

Reply:
agree, will update it.